oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/web/reactive/function/client/ServletBearerExchangeFilterFunction.java

```diff
@@ -61,14 +61,16 @@
 	}
 
 	private Mono<AbstractOAuth2Token> oauth2Token() {
-		return currentAuthentication()
+		return Mono.subscriberContext()
```

Comment:
A user looking at this code is going to wonder where the `Authentication` is being populated in the Context. We should javadoc to make it clear.